lib/OpenQA/WebAPI/Controller/API/V1/Worker.pm

```diff
@@ -177,15 +177,16 @@ sub create {
     my $validation = $self->validation;
     $validation->required($_) for qw(host instance cpu_arch mem_max worker_class);
     $validation->optional($_)
-      for qw(cpu_modelname cpu_opmode isotovideo_interface_version job_id websocket_api_version);
+      for qw(cpu_modelname cpu_opmode cpu_flags isotovideo_interface_version job_id websocket_api_version);
     return $self->reply->validation_error({format => 'json'}) if $validation->has_error;
 
     my $host     = $validation->param('host');
     my $instance = $validation->param('instance');
     my $job_ids  = $validation->every_param('job_id');
     my $caps     = {};
     $caps->{$_} = $validation->param($_)
-      for qw(cpu_arch cpu_modelname cpu_opmode mem_max isotovideo_interface_version websocket_api_version worker_class);
+      for
+      qw(cpu_arch cpu_modelname cpu_opmode cpu_flags mem_max isotovideo_interface_version websocket_api_version worker_class);
```

Comment:
I think we can do better for readability here :)

Reply:
Yes, but this how tidy want I write it.

Reply:
yes, but you could for example define a variable to use here or a list of line-wrapped constants at the top of the file.